documentation/jetty-documentation/src/main/asciidoc/programming-guide/websocket.adoc

```diff
@@ -28,6 +28,31 @@ Each side of a WebSocket connection, either client or server, is made of two ent
 * A xref:pg-websocket-endpoints[WebSocket _endpoint_], the entity that _receives_ WebSocket events.
 * A xref:pg-websocket-session[WebSocket _session_], the entity that offers an API to _send_ WebSocket data (and to close the WebSocket connection), as well as to configure WebSocket connection parameters.
 
+[[pg-websocket-configuration]]
+==== WebSocket Configuration
+
+There are various different ways to configure WebSocket connections, default configuration can be applied to the `WebSocketUpgradeFilter` and the `WebSocketContainer`, this default configuration is applied before the WebSocket is given to the application in the `onOpen()` notification, this configuration can further be modified in `onOpen()` and any time after using the configuration on the `Session`.
+
+Settings which can be configured include:
+
+* MAX_BINARY_MESSAGE_SIZE - The maximum size of a binary message which can be received. (default: 64 * 1024)
```

Comment:
These capital letters things are undefined.
You want to refer to explicit methods on the `Session` object, or otherwise a reader would have no idea _where_ to configure the maxBinaryMessageSize.

Same with the others below.

Also, no need of an empty line between the bullets.

See also the bullet list in the previous section: use:

* "definition", "explanation" 

don't use the single dash, if you want separation use either the command or the [em dash](https://www.thepunctuationguide.com/em-dash.html).

Reply:
@sbordet is there any way I can link java constants defined in `org.eclipse.jetty.websocket.core.WebSocketConstants` in case the defaults change?